src/js/edu-benefits/containers/SubmitPage.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import SubmitMessage from '../components/SubmitMessage';
+
+class SubmitPage extends React.Component {
+  render() {
+		// TODO: shared address component?
+    const address = (
```

Comment:
Looks like we'll be getting a `processedAt` property, so we'd probably use that somehow.